lang_GENERIC/parsing/lang.ml

```diff
@@ -72,6 +72,14 @@ let string_of_lang = function
   | ML -> "ML"
   | Go -> "Golang"
 
+(* Manually pulled from file_type_of_file2 in file_type.ml *)
+let ext_of_lang = function
```

Comment:
Can we make this automatic? if not this is ok

Reply:
It proved harder than I thought. WIthout doing refactor of file_type_of_file2 that uses some data structure, it seems it not possible to get the values from a pattern matching function!